spec/request/preneeds/cemeteries_request_spec.rb

```diff
@@ -3,6 +3,7 @@
 
 RSpec.describe 'Cemeteries Integration', type: :request do
   include SchemaMatchers
+  before(:each) { Redis.current.flushall }
```

Comment:
If this is tested in form_fill_spec.rb you could just load it with `allow_any_instance_of(Preneed::FormFill).to_receive(:new).and_return(FactoryGirl.build(:form_fill)` Which will be MUCH faster.